From 3babe54396c5043e0774278d4a0af4d1e29c769a Mon Sep 17 00:00:00 2001
From: Milind Changire <mchangir@redhat.com>
Date: Fri, 13 May 2016 00:54:25 +0530
Subject: [PATCH 143/158] packaging: additional dirs and files in /var/lib/glusterd/

Remove %config(noreplace) on /var/lib/glusterd/groups/virt

Correct spelling for
/var/lib/glusterd/ss_brick and
/var/lib/glusterd/snaps

Upstream:
> BUG: 1326410
> Gerrit-Url: http://review.gluster.org/14294

Change-Id: I6a744adc3088f6b6655bef07d361f51f21dcfb78
BUG: 1324820
Signed-off-by: Milind Changire <mchangir@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/74224
Reviewed-by: Kaleb Keithley <kkeithle@redhat.com>
Tested-by: Kaleb Keithley <kkeithle@redhat.com>
---
 glusterfs.spec.in |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/glusterfs.spec.in b/glusterfs.spec.in
index 60ffa65..c77754d 100644
--- a/glusterfs.spec.in
+++ b/glusterfs.spec.in
@@ -1306,7 +1306,7 @@ exit 0
        %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd
 %ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/bitd
        %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/groups
-            %attr(0644,-,-) %config(noreplace) %{_sharedstatedir}/glusterd/groups/virt
+            %attr(0644,-,-) %{_sharedstatedir}/glusterd/groups/virt
        %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/glusterfind
        %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/glusterfind/.keys
 %ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/glustershd
@@ -1356,8 +1356,8 @@ exit 0
 %ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/peers
 %ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/quotad
 %ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/scrub
-%ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/snap
-%ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/ss_bricks
+%ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/snaps
+%ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/ss_brick
 %ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/vols
 
 # Extra utility script
-- 
1.7.1

